torch/csrc/distributed/rpc/tensorpipe_utils.h

```diff
@@ -12,6 +20,76 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 
+#ifdef USE_CUDA_NOT_ROCM
+using at::cuda::CUDAStream;
+#endif
+
+
+struct FullDeviceContext {
```

Comment:
nit: Can we add some docs to this class?

Reply:
addressed